### DISCUSSION TITLE
LDAP authentification under forest multiple domain

### DISCUSSION THREAD

**Discussion by Intrudr-Sec** [authelia/authelia#5035] [General]
Hi,

I try to log with more than 1 domain as LDAP. If we have a.com and b.com, i want both user from those domain.
I have multiple linux services in places with LDAP config running on port 3268 (Global Catalog) and work fine.

One of them, I change the config on the base dn as "DC=a,DC=com;DC=b,DC=com"
So it take the first one if the user is create on both.

How can i do the same for Authelia ?

`  ldap:
    implementation: activedirectory
    url: ldap://192.168.0.19:3268
    start_tls: false
    tls:
      skip_verify: true
      minimum_version: TLS1.2
    base_dn: DC=a,DC=com
    username_attribute: sAMAccountName
    users_filter: (&(|({username_attribute}={input})({mail_attribute}={input}))(objectCategory=person)(objectClass=user)(!userAccountControl:1.2.840.113556.1.4.803:=2)(!pwdLastSet=0))
    #additional_groups_dn: OU=Groups
    groups_filter: (&(member:1.2.840.113556.1.4.1941:={dn})(objectClass=group)(objectCategory=group))
    group_name_attribute: cn
    mail_attribute: mail
    display_name_attribute: givenName
      #display_name_attribute: displayName
    user: CN=LDAP Query,OU=Test,DC=a,DC=com
    password: "aaabbbbcccdddeee"`

**Comment by james-d-elliott** [comment]
It's not currently possible I do not believe. We are trying to figure out the best way for us to tackle such a situation however and potentially other situations.
